api/controllers/CommentController.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
     )
   },
   createBatchFromEmailForm: function (req, res) {
+    // TODO: fix
```

Comment:
Fix for now or later?

Reply:
oh yeah this is an eventually someday if we ever bring back this feature... not really important enough to track with a ticket yet just wanted to flag in the code that this functionality is not totally working